### ISSUE TITLE
SEP-24: Remove 'Success: no additional information needed' section

### ISSUE THREAD

**Issue by JakeUrban** [stellar/stellar-protocol#539]
TL;DR: We should remove the `/interactive` [success](https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0024.md#1-success-no-additional-information-needed) response since we now have SEP-26 for non-interactive flows and we changed the `/interactive` endpoints from `GET` to `POST`.

Originally SEP-24's `/interactive` endpoints used to be `GET` requests, and had two different types of responses; one where more information is needed and another success response.

On the 'success' response:
> This is the correct response if the anchor is able to execute the withdrawal and needs no additional information about the user.

This sentence implies that SEP-24 anchors should support a non-interactive case, but SEP-26 was developed to address the need for non-interactive flows.

Furthermore, these endpoints were changed from `GET` requests to `POST` requests. `POST` requests should always be non-idempotent, but the SEP outlines non-idempotent behavior:
> It should also be used if the anchor requires information about the user, but the information has previously been submitted and accepted.

Thanks @andywer for pointing this out on stellar/django-polaris#149
